auth.go

```diff
@@ -105,7 +105,7 @@ func NewAutoAuth(login string, secret string) Authorizer {
 	})
 
 	az.AddAuthenticator("passport1.4", func(c *http.Client, rs *http.Response, path string) (auth Authenticator, err error) {
-		return NewPassportAuth(login, secret, rs.Request.URL.String(), &rs.Header)
+		return NewPassportAuth(login, secret)
```

Comment:
you should pass the `client` from here an perform the ping-pong auth.
because if we done here. we're going to serve the first request with `redo=true`